.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn pretty-quick --staged
+npm run pretty-quick --staged
```

Comment:
Can we stay with `yarn`? There have been some problems with `npm` on the CI recently

Reply:
Sure, changed it